src/components/screens/bookmarks/bookmarksList/bookmarksList.js

```diff
@@ -92,7 +95,11 @@ class BookmarksList extends React.Component {
 
   onTitleChange({ target }) {
     this.setState({
-      eddittedTitle: target.value,
+      ...this.state,
+      eddittedTitle: {
+        value: target.value,
```

Comment:
Error and feedback are conceptually the more similar than the input value. I think they should be grouped together.

Reply:
I dont understand, let me check it